src/__tests__/index.tsx

```diff
@@ -49,15 +49,9 @@ import * as React from 'react'
 export default ({children}) => <div className="sub-dir">{children}</div>
     `.trim(),
     },
-    rollup: {
-      getInputOptions(options) {
-        options.external = [...(options.external as Array<string>), 'left-pad']
-        return options
-      },
-      getOutputOptions(options) {
-        options.globals = {...options.globals, 'left-pad': 'myLeftPad'}
-        return options
-      },
+    esbuild: options => {
+      options.external = [...(options.external as Array<string>), 'left-pad']
```

Comment:
I've added a plugin that replicates rollups `global` option. I've also modified the test so that instead of just passing `left-pad` it now uses a custom function that works like left-pad but ignores the supplied filler (!) and uses $ instead. That way the test will fail if it uses the original `left-pad`

Reply:
Brilliant. I had thought about adding something like that after the PR (because I couldn't think how to explain what I wanted). Very good 👍

Reply:
I've commited the changes with this function in it now.